### ISSUE TITLE
Dev tools colorpicker (dropper) not working correctly

### ISSUE THREAD

**Issue by pixelass** [brave/browser-laptop#5944]
**Did you search for similar issues before submitting this one?**
Yes
**Describe the issue you encountered:**
The dropper only works inside the colorpicker.
**Expected behavior:**
The dropper should allow picking colors from the browser window or any other part of the screen.
- Platform (Win7, 8, 10? macOS? Linux distro?):
mac OS Sierra 10.12.1 (16B2555)
- Brave Version:
0.12.10
- Steps to reproduce:
    1. Open the web inspector
    2. Choose an element with a color property
    3. activate the colorpicker
    4. try picking colors outside the context

- Screenshot if needed:
<img width="310" alt="screen shot 2016-11-30 at 6 15 57 pm" src="https://cloud.githubusercontent.com/assets/1148334/20763350/f507dfaa-b729-11e6-9be6-cbecadeee3b5.png">

![dopper](https://cloud.githubusercontent.com/assets/1148334/20763356/f7f400d6-b729-11e6-9db1-852014096d8a.gif)

- Any related issues:

**Comment by pixelass** [comment]
I have no idea what just happened but the colorpicker has gone full haywire. This just happened when I switched back to the browser window. So the dripper does seem to work outside its context but not as intended. 

1. I am in fact targeting the part that is changing color
2. the colorpicker was still open
3. I switched between multiple tabs
4. It started working with malfunction after I returned to the tab

![dopper2](https://cloud.githubusercontent.com/assets/1148334/20764020/c9ef156a-b72c-11e6-8c27-e918f9aa02bb.gif)
